src/runtime.jl

```diff
@@ -125,79 +135,17 @@ Get the current task ID for this worker in hex format.
 """
 get_task_id() = String(ray_jll.Hex(ray_jll.GetCurrentTaskId(ray_jll.GetCoreWorker())))
 
-function parse_ray_args_from_raylet_out(session_dir)
-    #=
-    "Starting agent process with command: ... \
-    --node-ip-address=127.0.0.1 --metrics-export-port=60404 --dashboard-agent-port=60493 \
-    --listen-port=52365 --node-manager-port=58888 \
-    --object-store-name=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/sockets/plasma_store \
-    --raylet-name=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/sockets/raylet \
-    --temp-dir=/tmp/ray --session-dir=/tmp/ray/session_2023-08-14_14-54-36_055139_41385 \
-    --runtime-env-dir=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/runtime_resources \
-    --log-dir=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/logs \
-    --logging-rotate-bytes=536870912 --logging-rotate-backup-count=5 \
-    --session-name=session_2023-08-14_14-54-36_055139_41385 \
-    --gcs-address=127.0.0.1:6379 --minimal --agent-id 470211272"
-    =#
-    line = open(joinpath(session_dir, "logs", "raylet.out")) do io
-        while !eof(io)
-            line = readline(io)
-            if contains(line, "Starting agent process")
-                return line
-            end
-        end
-    end
+function get_node_to_connect_for_driver()
```

Comment:
Tests? I would definitely pass in `GLOBAL_STATE_ACCESSOR[]` and `NODE_IP_ADDRESS`.